### DISCUSSION TITLE
Jenkins email with Index.html for Serenity-BDD report does not have Summary images

### DISCUSSION THREAD

**Discussion by nkalisamy** [serenity-bdd/serenity-core#3057] [Q&A] [answered]
Hi All,

I am currently using Jenkins for sending the report via email to the team and eventually got it to working with Extended Email Notification option. With this, the index.html is sent to the relavant parties. 
However, the index.html when its sent as a separate file, I could only see plain html file and the supporting files are not loaded which in turn shows just the text. 

1. Is there a way to include Summary as in Passed or Fail in the index file instead of Images(Charts) 

OR

2. Is there a way to send the index.html file with the supporting file included or embedded? So that the when we open the index.html file is opened with charts like its showing with local files?  

If there is any way for the 2nd then thats most preferred because as we know the report is just excellent and pleasant to see.

I tried to lookout for an option for past one day but still stuck with it.

Please let me know if anyone has done this, thanks/

**Comment by wakaleo** [comment, marked as answer]
You can't send the index.html as it relies on JS libraries to work and won't rended in an email. For email notifications, use the single page report (see https://github.com/serenity-bdd/serenity-core/tree/main/serenity-single-page-report)

**Reply by nkalisamy** [reply]
Thanks very much @wakaleo for the quick response. I just verified it and single-page report should be fine with me.
